packages/theme/src/client/module/navbar/components/icons/repoIcon.ts

```diff
@@ -42,7 +42,7 @@ BitbucketIcon.displayName = "BitbucketIcon";
 export const SourceIcon: FunctionalComponent = () =>
```

Comment:
Here? Also I think just keeping the original icon name should be fine.

Reply:
Ok, I'll fix it soon. Just wait a moment, I have some problems with my IDE. Webstorm frequently crashed after updating to 2022.1. i will roll back to 2021.

Reply:
Do you mean to just replace svg content? Or to replace the "SourceIcon" with a new "ForkIcon".